### DISCUSSION TITLE
Unable to set custom icon at OpenID connect

### DISCUSSION THREAD

**Discussion by gmentzik** [Budibase/budibase#10771] [Help] [answered]
I am on free self host.
I am unable to set custom icon at OpenID Connect authentication. 
Tried on both 2.5 and 2.6 version. Icon appears corrupted.

Should the icon be saved and sized in a special way in order to be correctly loaded?
![image](https://github.com/Budibase/budibase/assets/65217733/36c01443-f686-47bf-97d3-9c54a6a82c49)

![image](https://github.com/Budibase/budibase/assets/65217733/124532c3-faac-4978-8130-4e79ea8c9cd3)

**Comment by ConorWebb96** [comment, marked as answer]
Hey @gmentzik,

There was a fix for this, however it only seemed to look at the default icons; https://github.com/Budibase/budibase/pull/9826. I've created a new issue for custom icons, comment under the issue created as it will help with prioritising this to get it fixed sooner. https://github.com/Budibase/budibase/issues/10812

Thanks highlighting this, hopefully this helps!
